pkgs/build-support/ocaml/default.nix

```diff
@@ -1,20 +1,25 @@
 { lib, stdenv, writeText, ocaml, findlib, ocamlbuild, camlp4 }:
 
-{ name, version, nativeBuildInputs ? [],
-  createFindlibDestdir ?  true,
-  dontStrip ? true,
-  minimumSupportedOcamlVersion ? null,
-  hasSharedObjects ? false,
-  setupHook ? null,
-  meta ? {}, ...
+{ name
+, version
+, nativeBuildInputs ? [ ]
+, createFindlibDestdir ? true
+, dontStrip ? true
+, minimumOCamlVersion ? null
```

Comment:
Can we have the old attribute throwing an error when set with the hint to switch to the new attribute name?
This would help nixpkgs user a lot.

Reply:
sure we could do this. I thought more of an documentation only and not have code that gets removed in the future.